app/code/core/Mage/Oauth/Block/Adminhtml/Oauth/AuthorizedTokens/Grid.php

```diff
@@ -206,9 +204,7 @@ public function decorateUserType($value, $row, $column, $isExport)
     public function decorateUserId($value, $row, $column, $isExport)
     {
         $value = ($row->getCustomerId())   ?$row->getCustomerId()   :$row->getAdminId();
-        $cell = $value;
-
-        return $cell;
+        return $value;
```

Comment:
can I suggest to replace by `return $row->getCustomerId() ?? $row->getAdminId();`